js/client.js

```diff
@@ -374,11 +374,21 @@
 			this.topbar = new Topbar({el: $('#header')});
 			if (this.down) {
 				this.isDisconnected = true;
-			} else if (document.location.hostname === 'play.pokemonshowdown.com' || Config.testclient) {
-				this.addRoom('rooms', null, true);
+			} else {
+				if (document.location.hostname === 'play.pokemonshowdown.com' || Config.testclient) {
+					this.addRoom('rooms', null, true);
+				} else {
+					this.addRoom('lobby', null, true);
+				}
 				Storage.whenPrefsLoaded(function () {
-					var autojoin = (Tools.prefs('autojoin') || '');
+					var autojoin = (Tools.prefs('autojoin') || {});
 					var autojoinIds = [];
+					if (typeof autojoin === 'string') {
+						// Switch to the new autojoin object
+						autojoin = {showdown: autojoin};
```

Comment:
Reason I did that is so it uses the new object based autojoin system if your cookies are storing the old string based one still. I could run a setup so that if you don't already have a object based setup and its PS main your connecting too, to just use a string if you want.